docs/configuration-reference/components/metallb.md

```diff
@@ -77,6 +77,9 @@ component "metallb" {
 
 MetalLB will use the specified CIDR for exposing services of type `LoadBalancer`.
 
+>NOTE: The official MetalLB [documentation](https://metallb.universe.tf/) contains information
```

Comment:
This links to the main website. Is that the intention ? or to a specific page ?

Reply:
I've deliberately linked to the main page. What other page did you have in mind?

Reply:
I meant if the link to the main website was deliberate or accidental. If deliberate, no issues :+1: